chat-agent/tests/test_integration.py

```
@@ -15,803 +15,416 @@
+from unittest.mock import AsyncMock, MagicMock, patch
+
+from fastapi.testclient import TestClient
+from httpx import AsyncClient, ASGITransport
+
+from api.http import app, MetricsTracker
+
+# A minimal valid JWT whose payload decodes to {"userid": "test-user"}
+_SAMPLE_TOKEN = (
+    "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9."
+    "eyJ1c2VyaWQiOiJ0ZXN0LXVzZXIifQ."
+    "SflKxwRJSMeKKF2QT4fwpMeJf36POk6yJV_adQssw5c"
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "_SAMPLE_TOKEN"

(S105)